test/unit/spec/MfaVerify_spec.js

```diff
@@ -2301,6 +2305,76 @@ function (Okta,
                 });
               });
             });
+            itp('starts poll after a delay of 6000ms', function () {
+              return setupOktaPush().then(function (test) {
+                spyOn(LoginUtil, 'callAfterTimeout').and.callFake(function() {
+                  // reducing the timeout to 100 so that test is fast.
+                  return setTimeout(arguments[0], 100);
+                });
+                $.ajax.calls.reset();
+                test.setNextResponse(resChallengePush);
+                test.form[0].submit();
+                return tick(test).then(function() {
+                  expect(LoginUtil.callAfterTimeout.calls.argsFor(1)[1]).toBe(6000);
+                  var deferred = Q.defer();
+                  expect(test.router.controller.model.appState.get('transaction').status).toBe('MFA_CHALLENGE');
+                  var transaction = test.router.controller.model.appState.get('transaction');
+                  spyOn(transaction, 'poll');
+                  expect(transaction.poll).not.toHaveBeenCalled();
```

Comment:
is this needed? If you just spied on `poll` on the line before, then there is no other place where this would be called, right?